### PR TITLE
✨ Add MongoDB and RegisterPage function

### PR DESCRIPTION
**Pull request by dhnazarenko**

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

File: src/server.ts [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
-  console.log(`Server listening at http://localhost:${PORT}`);
-});
-
 // Handle client routing, return all requests to the app
 app.get('*', (_req, res) => {
   res.sendFile(path.join(__dirname, 'app/index.html'));
 });
+app.get('/', (_req, res) => {
+  res.send('Hello World!');
+});
```
Comment (lajanzen) [inline src/server.ts]: Do you need this?

**Review by lajanzen — APPROVED** [review]
🚀

File: src/server.ts (hunk 1 above)
Comment (dhnazarenko) [reply src/server.ts]: Thanks! I took it out.

**Pull request merged**